### DISCUSSION TITLE
Change babel require line

### DISCUSSION THREAD

**Discussion by kotmatpockuh** [exceljs/exceljs#2518] [Ideas & Proposals]
Please change it to `require('@babel/polyfill');` in `exceljs.nodejs.js`

ps: for v2

**Comment by Siemienik** [comment]
@guyonroche @alubbe Do you know anything about it? Could you help?

**Comment by guyonroche** [comment]
@kotmatpockuh The main export from ExcelJS is supports Node v8 and above (i.e. es2016). These versions of node do not need the polyfills.
If you need to use an earlier version of node, you can use 'exceljs/dist/es5' and as explained here [ES5 Imports](https://github.com/exceljs/exceljs#es5-imports) you will need to require "core-js" and "regenerator-runtime" to your dependencies.
